system/Commands/Utilities/Routes/AutoRouteCollector.php

```diff
@@ -35,7 +35,7 @@ public function __construct(string $namespace, string $defaultController, string
     }
 
     /**
-     * @return list<list<string>>
+     * @return array<int, array<int, string>>
```

Comment:
PHPStan does not fully support `list`. However, there should be no error.
https://github.com/phpstan/phpstan/issues/6358#issuecomment-1012168140

Reply:
Hmm. I did not know that for `list`.